From 201068e90ebe63f07866108d375291d243f27c56 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Wed, 29 Jun 2016 12:17:22 +0300
Subject: [PATCH 0328/1345] dts: support tx_interrupts for 4 CPU's

commit  b101badcfbe0fb1ecdbc061f371ce53de462684e from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- interrupt number align to u-boot interrupts

Change-Id: Iab404129aaf01521af5c3b87202024da3e4da46b
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30828
Reviewed-by: Neta Zur Hershkovits <neta@marvell.com>
Tested-by: Neta Zur Hershkovits <neta@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30901
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi |    6 +++---
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi   |    6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
index 7e577b8..d9956f6 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
@@ -250,7 +250,7 @@ ppv22@000000 {
 			     <GIC_SPI 265 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#1 Interrupt */
 			     <GIC_SPI 269 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#2 Interrupt */
 			     <GIC_SPI 273 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#3 Interrupt */
-			     <GIC_SPI 277 IRQ_TYPE_LEVEL_HIGH>; /* RX Shared Interrupt */
+			     <GIC_SPI 316 IRQ_TYPE_LEVEL_HIGH>; /* RX Shared Interrupt */
 		port-id = <0>; /* pp2_port_id */
 		emac-data = <&emac0_1>;
 		status = "disabled";
@@ -260,7 +260,7 @@ ppv22@000000 {
 			     <GIC_SPI 266 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#1 Interrupt */
 			     <GIC_SPI 270 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#2 Interrupt */
 			     <GIC_SPI 274 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#3 Interrupt */
-			     <GIC_SPI 278 IRQ_TYPE_LEVEL_HIGH>; /* RX Shared Interrupt */
+			     <GIC_SPI 317 IRQ_TYPE_LEVEL_HIGH>; /* RX Shared Interrupt */
 		port-id = <1>; /* pp2_port_id */
 		emac-data = <&emac2_1>;
 		status = "disabled";
@@ -270,7 +270,7 @@ ppv22@000000 {
 			     <GIC_SPI 267 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#1 Interrupt */
 			     <GIC_SPI 271 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#2 Interrupt */
 			     <GIC_SPI 275 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#3 Interrupt */
-			     <GIC_SPI 279 IRQ_TYPE_LEVEL_HIGH>; /* RX Shared Interrupt */
+			     <GIC_SPI 318 IRQ_TYPE_LEVEL_HIGH>; /* RX Shared Interrupt */
 		port-id = <2>; /* pp2_port_id */
 		emac-data = <&emac3_1>;
 		status = "disabled";
diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index 97a597c..777f92b 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -325,7 +325,7 @@ ppv22@000000 {
 			     <GIC_SPI 41 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#1 Interrupt */
 			     <GIC_SPI 45 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#2 Interrupt */
 			     <GIC_SPI 49 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#3 Interrupt */
-			     <GIC_SPI 53 IRQ_TYPE_LEVEL_HIGH>; /* RX Shared Interrupt */
+			     <GIC_SPI 92 IRQ_TYPE_LEVEL_HIGH>; /* RX Shared Interrupt */
 		port-id = <0>; /* pp2_port_id */
 		emac-data = <&emac0>;
 		status = "disabled";
@@ -335,7 +335,7 @@ ppv22@000000 {
 			     <GIC_SPI 42 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#1 Interrupt */
 			     <GIC_SPI 46 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#2 Interrupt */
 			     <GIC_SPI 50 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#3 Interrupt */
-			     <GIC_SPI 54 IRQ_TYPE_LEVEL_HIGH>; /* RX Shared Interrupt */
+			     <GIC_SPI 93 IRQ_TYPE_LEVEL_HIGH>; /* RX Shared Interrupt */
 		port-id = <1>; /* pp2_port_id */
 		emac-data = <&emac2>;
 		status = "disabled";
@@ -345,7 +345,7 @@ ppv22@000000 {
 			     <GIC_SPI 43 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#1 Interrupt */
 			     <GIC_SPI 47 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#2 Interrupt */
 			     <GIC_SPI 51 IRQ_TYPE_LEVEL_HIGH>, /* TX CPU#3 Interrupt */
-			     <GIC_SPI 55 IRQ_TYPE_LEVEL_HIGH>; /* RX Shared Interrupt */
+			     <GIC_SPI 94 IRQ_TYPE_LEVEL_HIGH>; /* RX Shared Interrupt */
 		port-id = <2>; /* pp2_port_id */
 		emac-data = <&emac3>;
 		status = "disabled";
-- 
1.7.9.5

